Documentation/markdown/SetupHelp.md

```diff
@@ -35,6 +39,14 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
 
  Clone **OpenInfraPlatform** repository  - more information you can find [here](./GitProcess.md)
 
+ Get Dot and Doxygen:
+
+	 1. After cloning **OpenInfraPlatform** repository open OpenInfraPlatform source folder. 
+	 2. Open folder **external**
+	 3. Open files:
+		* GetDot.cmd  
+		* GetDoxygen.cmd
+
```

Comment:
This belongs in a separate section, called "Generating documentation" at the end of the document.

Reply:
Resolved: 439c61b